.gitignore

```diff
@@ -20,3 +20,6 @@ vendor/
 
 .idea/
 log/
+
+benchmark-*.error
```

Comment:
To get rid of generated files in the repo when testing the benchmark.sh script locally